CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v1.6.0
+
+**The minimum system requirements for this version of Nerves have changed**
+Nerves 1.6.0 requires at least Elixir v1.7.0 and Erlang/OTP 21. If your
+project requires an older version of Elixir or Erlang/OTP you can pin
+the version of nerves and nerves_bootstrap to an older version.
```

Comment:
For example, in your `mix.exs` set your `nerves` dependency to:

```elixir
    {:nerves, "~> 1.5.0", runtime: false},
```

To force `nerves_bootstrap`, run:

```elixir
mix archive.install hex nerves_bootstrap 1.7.0
```